docs/Developers/Development_Setup.rst

```diff
@@ -52,6 +52,10 @@ Using virtualenv
         $ virtualenv coala-venv
 
 NOTE:
+It is recommended that you install virtualenv with sudo rights
```

Comment:
Recommended is the wrong word here, since it's almost certainly a requirement. I'd say "Depending on your system, you most likely need `sudo` to innnstall virtualenv"